### DISCUSSION TITLE
ChoiceVar: Filter on a custom field

### DISCUSSION THREAD

**Discussion by jvanderaa** [nautobot/nautobot#611] [Q&A]
Looking to filter a ChoiceVar inside of a Job based on some data in a custom field. This is going to be an optional custom field, that I would like to get all of the objects that have `True` or `False`, skipping the objects that do not have that defined. Any immediate thoughts?

**Comment by jathanism** [comment]
Since you're talking about using a `Job`, is it safe to assume you're using the ORM models? This *should* be feasible using the ORM. Got some example data we can muck with to get you the answer you need?
